src/Sylius/Component/Channel/Model/Channel.php

```diff
@@ -24,32 +24,32 @@ class Channel implements ChannelInterface
     use TimestampableTrait, ToggleableTrait;
 
     /**
-     * @var mixed
+     * @var mixed|null
      */
     protected $id;
 
     /**
-     * @var string
+     * @var string|null
```

Comment:
I'm OK with both, the only reason for using `|null` is that there are more occurrences of this one in our code and that means... shorter diffs :) Anyway, we can discuss to replace them project-wide (again, with a static analysis tool configured to enforce that).